web/src/main/java/se/inera/intyg/webcert/web/service/facade/util/IntygToCertificateConverterImpl.java

```diff
@@ -140,6 +140,9 @@ private Certificate convertToCertificate(IntygContentHolder certificate) {
             intygTextsService.isLatestMajorVersion(certificateToReturn.getMetadata().getType(),
                 certificateToReturn.getMetadata().getTypeVersion())
         );
+        certificateToReturn.getMetadata().setAvailableForCitizen(
+            !(certificate.getUtlatande().getTyp().equals("db") || certificate.getUtlatande().getTyp().equals("doi"))
```

Comment:
You can use the constants in DoiModuleEntryPoint and DbModuleEntryPoint instead of hardcoding it.

Reply:
will do